apps/gateway/lib/gateway/controllers/auth_controller.ex

```diff
@@ -0,0 +1,21 @@
+defmodule Gateway.Controllers.AuthController do
+  @moduledoc """
+  Controller for users authentication.
+  """
+  use Gateway, :controller
+  alias Gateway.Auth.GoogleTokenManager
+  alias GameBackend.Users
+
+  def validate_token(conn, %{"token_id" => token_id}) do
+    conn = put_resp_content_type(conn, "application/json")
```

Comment:
Is this really necessary? Cause I'm pretty sure its not

Reply:
Yeah, its not. Deleted!